local-config/collector.config.yaml

```diff
@@ -13,7 +13,7 @@ exporters:
     loglevel: debug
 
   otlp/1:
-    endpoint: host.docker.internal:21321
+    endpoint: tracetest:21321
```

Comment:
this was probablly a leftover from someone doing docker tests, it should point to the internal url

Reply:
Question: is this file intended just for local testing, or do we use it in some pipelines?

Reply:
local testing. It's used when you run the `./run.sh` script for example, that is mainly targetted to run an env similar to CI, so you can run the tracetesting tests locally easily.